docs/src/man/latex.md

```diff
@@ -36,6 +36,8 @@ This is the binomial coefficient.
 func(x) = # ...
 ```
 
+Do note however that the syntax using double-backticks described below is preferred over the traditional dollar-syntax for math expressions.
```

Comment:
Shouldn't we just update the examples in this file to use double backticks instead?

Reply:
Not sure, then the whole point of the "escaping" section is moot, no? And I assume that a lot of people will still use `$...$` out of habit.

Reply:
No, you still need to escape the `\` in `\frac` etc.

Reply:
True. I'll just update it, then.